cardshifter-server/src/test/java/com/cardshifter/server/main/MainServerTest.java

```diff
@@ -0,0 +1,27 @@
+package com.cardshifter.server.main;
+
+import com.cardshifter.server.model.MainServer;
+import com.cardshifter.server.model.Server;
+import org.apache.log4j.PropertyConfigurator;
+import org.junit.Test;
+
+/**
+ * Created by marc on 2017-02-07.
+ */
+public class MainServerTest {
+
+    @Test(expected = IllegalStateException.class)
+    public void testThrowWhenNoModAreLoaded() {
```

Comment:
Normally it loads them from the build.gradle with the workdir line. It loads the mods from extra-ressources. Since we load the defaults automatically in the initialization of the class, I had to refactor it in the constrcutor to not load the mods in the test.